docs/links.md

```diff
@@ -1,31 +1,46 @@
 # Links
 
-This file contains links to files not stored in this repository.
-Many of them are drafts in Google docs for better collaboration experience.
+This file contains links to files **not stored in this repository**. See the
+individual [projects](../README.md#projects) for docs relevant to those
+projects.
 
-## Access to the docs
-
-Some documents listed are shared within limited google groups, 
-please at least join one of the following groups to access the docs:
+Many of files listed below are drafts in Google docs for better collaboration
+experience.  Some of these are shared within limited google groups, please at
+least join one of the following groups to access the docs:
 
 - [kubernetes-dev](https://groups.google.com/forum/#!forum/kubernetes-dev)
 - [kubernetes-wg-multitenancy](https://groups.google.com/forum/#!forum/kubernetes-wg-multitenancy)
 
+## Working Group Meeting Records
+
+- [YouTube Channel](https://www.youtube.com/playlist?list=PL69nYSiGNLP1tBA0W8zEe6UwPsabGQk-j&disable_polymer=true)
+- [Meeting Notes and Agenda](https://docs.google.com/document/d/1fj3yzmeU2eU8ZNBCUJG97dk_wC7228-e_MmdcmTNrZY/edit)
+
+## Presentations
+
+- Kubecon EU 2020 ("Amsterdam" or wherever COVID dropped you, August)
+  - Multi-tenant clusters with hierarchical namespaces (@adrianludwin, [Aug 19](https://sched.co/Zeuh))
```

Comment:
Do you mind adding vc talk as well? :)

- Virtual Cluster - A Practical Kubernetes Hard Multi-tenancy Solution (@Fei-Guo, [Aug 19](https://sched.co/Zek6))

Reply:
Done!